lib/master.go

```diff
@@ -213,10 +213,6 @@ func (m *DefaultBaseMaster) Init(ctx context.Context) error {
 }
 
 func (m *DefaultBaseMaster) doInit(ctx context.Context) (isFirstStartUp bool, err error) {
-	if err := m.registerMessageHandlers(ctx); err != nil {
-		return false, errors.Trace(err)
-	}
-
 	isInit, epoch, err := m.refreshMetadata(ctx)
```

Comment:
this panic will happen when the goroutine run  between  the line 219 and line 234. and a heartbeat arrives?

Reply:
Yes